script.js

```diff
@@ -6,13 +6,120 @@ const menuIcon = document.querySelector('.menuIcon');
 const menuItem = document.querySelector('.menu-item-desktop');
 const items = document.querySelectorAll('.menuItemDesktop');
 
+const body = document.querySelector('body');
+const cardData = [
+  {
+    name: 'Project 1',
+    description:
+      'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsam nobis nemo vitae tempora et magni in corporis earum illum cupiditate repudiandae, amet nihil reprehenderit voluptatum tempore eligendi deserunt id adipisci.',
```

Comment:
- [ ] Great job so far team :100: I will suggest that you increase the font size of your popup paragraph text. Currently, it looks smaller than in the Figma design :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/232047153-e34914b4-a6e6-402a-99fe-3234e075ef04.png)

- The Figma design

| Figma Design | Styles |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/232047076-b1e4a7c9-8358-49fa-8223-374abcbfb4cb.png) | ![image](https://user-images.githubusercontent.com/102699958/232046979-8d85a80c-3e6b-4496-b4f3-c949d37fd1e9.png) |